README.md

```diff
@@ -17,8 +17,9 @@ The motivation for this is similar to ngReact's:
 - [Install](#install)
 - [Usage](#usage)
 - [Features](#features)
-	- [the vue-component directive](#the-vue-component-directive)
-	- [the createVueComponent factory](#the-createvuecomponent-factory)
+	- [The vue-component directive](#the-vue-component-directive)
+	- [The createVueComponent factory](#the-createvuecomponent-factory)
+	- [Event Handling](#event-handling)
```

Comment:
I would not add this intel as a feature, at the same level of the `vue-component` directive and `createVueComponent` factory. It works thanks to `vprops-*`, so I think it'd better be right after this piece of code: 
```javascript
<vue-component vprops="ctrl.person" />
// equals to
<vue-component vprops-first-name="ctrl.person.firstName" vprops-last-name="ctrl.person.lastName" />
```
And a little bit shorter (the whole bunch of code I wrote in the issue is slightly opinionated). I'd put it this way:
___
Events can bubble up from Vue to AngularJS components by binding functions references as `vprops-*`
```javascript
<vue-component vprops-on-click="ctrl.handleClick"></vue-component>
```
```javascript
const VComponent = Vue.component('hello-component', {
  props: {
    onClick: Function
  },
  render (h) {
    return (
      <button onClick={this.onClick}></button>
    )
  }
})
```